### ISSUE TITLE
squad_model_noans для русского языка планируется?

### ISSUE THREAD

**Issue by trembolon58** [deepmipt/DeepPavlov#683]
Здравствуйте, без возможности определить на сколько SQuAD ответ соответствует контексту, практическое использование наработок проблематично.

**Comment by yurakuratov** [comment]
Добрый день, для русского языка пока можно воспользоваться моделью squad_ru и выходом `logits`, чем больше значение, тем сильнее модель уверена в ответе.

Модель `multi_squad_model_noans` для русского на текущий момент сделать проблематично, так как нет данных, аналогичных тем, на чем тренировалась `multi_squad_model_noans` модель.

`squad_ru` модель обучена на SDSJ Task B датасете и в нем, по сравнению со SQuAD, неизвестно откуда был взят фрагмент текста для ответа.

**Comment by trembolon58** [comment]
Не подскажете, как нормализовать logits от 0 до 1? Методом проб и ошибок не удалось выявить пороговое значение. Часто logits у ошибочного ответа больше в 2 раза по сравнению с значением верного.

**Comment by yurakuratov** [comment]
не могу предложить хорошего решения, кроме как подбора порога на некоторой отложенной выборке

**Comment by trembolon58** [comment]
Можете скинуть ссылку на датасет и написать небольшой туториал, как его модифицировать для поддержки noans?

**Comment by yurakuratov** [comment]
Ссылка на датасет для английского есть вот [тут](https://github.com/deepmipt/DeepPavlov/blob/fd7e8504504e0d4e2d72cb857e95c2406107c921/deeppavlov/dataset_readers/squad_dataset_reader.py#L44).
Датасет для русского должен быть в таком же формате, что и для английского и содержать:
1. Контексты с ответом из датасета SDSJ Task B
2. Контексты из статей Википедии с разметкой ответов (можно разметить все вхождения подстрок равных ответу на вопрос)

Контексты из Википедии для каждого вопроса можно извлечь с помощью ранкера документов
с конфигом `ru_ranker_tfidf_wiki.json` из
https://github.com/deepmipt/DeepPavlov/tree/master/deeppavlov/configs/doc_retrieval

**Comment by yoptar** [comment]
Закрываю за неактивностью темы.
